### ISSUE TITLE
Set pref("clipboard.autocopy", false) to true

### ISSUE THREAD

**Issue by aDifferentBook** [Divested-Mobile/Mull-Fenix#38]
Started using Mull Browser and found I can't copy links from sites and after searching I found [Issue #24](https://github.com/Divested-Mobile/Mull-Fenix/issues/24) to solve my issue. 

But for many other people it's hard to find things and configure them so it's better to enable it by default.

I've set these two values to true -

```
pref("clipboard.autocopy", false); - true
pref("dom.allow_cut_copy", false); - true
```

**Comment by SkewedZeppelin** [comment]
I'll consider it for next version

**Comment by SkewedZeppelin** [comment]
This is fixed in the latest 98.1.1 release.
